oonib/report/handlers.py

```diff
@@ -192,27 +140,63 @@ def parseNewReportRequest(request):
 
     return parsed_request
 
-class ReportHandler(OONIBHandler):
+def closeReport(report_id):
+    report_dir = FilePath(config.main.report_dir)
+    archive_dir = FilePath(config.main.archive_dir)
+    report_path = report_dir.child(report_id)
+    report_metadata_path = report_dir.child(report_id +
+                                            "-metadata.json")
+    print "Checking if %s exists and %s exists" % (report_path,
+                                                   report_metadata_path)
+    if not report_path.exists() or \
+            not report_metadata_path.exists():
+        raise e.ReportNotFound
+
+    with report_metadata_path.open('r') as f:
+        report_details = json.load(f)
```

Comment:
I see that the case where `json.load()` throws `ValueError` because the metadata file is corrupted is not handled. Do we want to handle this case? (I think it might happen if we close when writing a metadata file is in progress.)